hcloud/iso.go

```diff
@@ -60,22 +60,11 @@ func (c *ISOClient) GetByID(ctx context.Context, id int) (*ISO, *Response, error
 
 // GetByName retrieves an ISO by its name.
 func (c *ISOClient) GetByName(ctx context.Context, name string) (*ISO, *Response, error) {
-	path := "/isos?name=" + url.QueryEscape(name)
-	req, err := c.client.NewRequest(ctx, "GET", path, nil)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	var body schema.ISOListResponse
-	resp, err := c.client.Do(req, &body)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	if len(body.ISOs) == 0 {
-		return nil, resp, nil
+	isos, response, err := c.List(ctx, ISOListOpts{Name: name})
+	if len(isos) > 0 {
```

Comment:
Sometimes it’s `if len(...) == 0`, sometime it’s `if len(...) > 0`. Should be consistent across all clients.